app/services/UIBuilder.scala

```diff
@@ -196,4 +197,14 @@ class UIBuilder {
   private def fromSectionAndNoteCallout(caption: Text, nc: NoteCallout)(implicit stanzaIdToUrlMap: Map[String, String], lang: Lang): UIComponent =
     Details(caption, Seq(TextBuilder.fromPhrase(nc.text)))
 
+  private def fromSequence(sequence: Sequence, components: Seq[UIComponent])
```

Comment:
Looking at GDS, it supports hints on each checkbox item, so we'll need to support the [hint:....] label at the end of each checkbox item text as the main text

Reply:
I have raised separate task for this (EG-1309)